http-server/src/main/java/com/facebook/airlift/http/server/HttpServerModule.java

```diff
@@ -80,4 +81,10 @@ public void configure(Binder binder)
 
         binder.bind(AnnouncementHttpServerInfo.class).to(LocalAnnouncementHttpServerInfo.class).in(Scopes.SINGLETON);
     }
+
+    @Provides
+    public Authorizer getAuthorizer(Authorizer authorizer)
+    {
+        return authorizer;
```

Comment:
Is this going to be `null` by default?

Reply:
Generally we don't bind null instances. We should figure out a better way to have a default.